tests/snapshots.rs

```diff
@@ -154,7 +154,7 @@ fn check_output_msl(
     let options = &params.msl;
     #[cfg(not(feature = "deserialize"))]
     let options = if params.msl_custom {
-        println!("Skipping {}", destination);
+        println!("Skipping {}", destination.display());
```

Comment:
why is this calling `display()`?

Reply:
A path can't be printed without called `.display()`. I'm not sure how this worked originally.

Reply:
But... `{}` *is* supposed to call `display`. What am I missing here?

Reply:
`{}` calls `<T as Display>::fmt`. Paths don't implement that directly because they can contain invalid unicode, so you have to call `.display()` to convert them into something that can be printed.

Reply:
ok, so I'm still unsure how this worked :)
that is not a blocker for this to land